### DISCUSSION TITLE
Add support for Modded Forks

### DISCUSSION THREAD

**Discussion by Liopoxys** [BG-Software-LLC/SuperiorSkyblock2#1424] [Pending Feature Requests] [answered]
I would like to request plugin to make it work with modded forks like magma/arclight! there are a lot of owners that would love to use SB with mods!

**Comment by OmerBenGera** [comment, marked as answer]
Nothing to do on my end.
The plugin uses NMS, and if they modify the NMS without keeping backwards compatibility, then nothing can be done regarding this.
